frame/staking/src/tests.rs

```diff
@@ -3461,6 +3461,75 @@ mod offchain_phragmen {
 		})
 	}
 
+	#[test]
+	fn nomination_slash_filter_is_checked() {
+		// If a nominator has voted for someone who has been recently slashed, that particular
```

Comment:
> that particular nomination should be disabled for the upcoming election.

Does "nomination" refer to the entire set of approval votes that a nominator casts, or the individual approval vote(s) for recently-slashed validators.

For example, imagine Norman is a nominator, and Victor and Vanessa are validators. Normal casts approval votes for both Victor and Vanessa. However Victor was recently slashed. Which of these is correct?

A) Norman's entire nomination should be disabled because he voted for a single recently-slashed validator.
B) Norman's vote for Victor should be disabled, but his vote for Vanessa is valid and should be included.

(Also, typo: mut -> must)

Reply:
> Does "nomination" refer to the entire set of approval votes that a nominator casts, or the individual approval vote(s) for recently-slashed validators.



Precisely individual, so B is correct.